src/arrayReverse.test.js

```diff
@@ -8,13 +8,38 @@ describe(`Function 'arrayReverse':`, () => {
   });
 
   it(`should return an array`, () => {
-
+    expect(arrayReverse(['Mate', 'Academy'])).toBeInstanceOf(Array);
   });
 
   it(`should return an empty string
     if original array consists of an empty string`, () => {
-    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
+    expect(arrayReverse([''])).toEqual(['']);
+  });
+
+  it(`should reverse the words`, () => {
+    expect(arrayReverse(['Mate'])).toEqual(['etaM']);
+  });
+
+  it(`should preserve the length of each word`, () => {
+    const initialArrary = ['Mate', 'Academy'];
+    const reversedArray = arrayReverse(initialArrary);
+
+    expect(initialArrary[0].length).toBe(reversedArray[0].length);
+    expect(initialArrary[1].length).toBe(reversedArray[1].length);
   });
 
-  // write more tests here
+  it(`should return the same amount of the words`, () => {
```

Comment:
```suggestion
  it(`should return the same number of the words`, () => {
```
https://www.bkacontent.com/gs-commonly-confused-words-amount-number-and-quantity/